src/components/tabs/Tabs.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Tab } from '../tab';
+import { TabsShape } from '../shapes/TabsShape';
+import './Tabs.css';
+
+export class Tabs extends React.PureComponent {
+  state = {
+    activeIndex: this.props.index,
```

Comment:
```suggestion
    activeIndex: 0,
```

the default index is always 0